sibilant/pseudops/__init__.py

```diff
@@ -564,6 +566,7 @@ def reset(self):
         self.blocks = [CodeBlock(Block.BASE, 0, 0)]
         self.consts = [None]
 
+        self.async = False
```

Comment:
this may have to be come async_func or asynchronous

Reply:
I renamed it to `coroutine`, which lines up a little better with the language in [PEP 492](https://www.python.org/dev/peps/pep-0492/) (especially since we don't require the `async` marking keyword anywhere) and the name of the flag.